skrub/tests/test_table_vectorizer.py

```diff
@@ -642,6 +642,95 @@ def test__infer_date_format() -> None:
     assert _infer_date_format(date_column) is None
 
 
+@pytest.mark.parametrize(
+    ["specific_transformers", "expected_transformers_"],
+    [
+        (
+            (MinHashEncoder(), ["str1", "str2"]),
+            [
+                ("numeric", "passthrough", ["int", "float"]),
+                ("minhashencoder", "MinHashEncoder", ["str1", "str2"]),
+                ("low_card_cat", "OneHotEncoder", ["cat1", "cat2"]),
+            ],
+        ),
+        (
+            ("mh_cat1", MinHashEncoder(), ["cat1"]),
+            [
+                ("numeric", "passthrough", ["int", "float"]),
+                ("mh_cat1", "MinHashEncoder", ["cat1"]),
+                ("low_card_cat", "OneHotEncoder", ["str1", "str2", "cat2"]),
+            ],
+        ),
+    ],
+)
+def test_specifying_specific_column_transformer(
+    specific_transformers, expected_transformers_
+) -> None:
+    """
+    Test that specifying a specific transformer works as expected.
+    """
+    X = _get_dirty_dataframe()
+    tv = TableVectorizer(specific_transformers=[specific_transformers]).fit(X)
+    clean_transformers_ = [
+        (name, transformer.__class__.__name__, columns)
+        if not isinstance(transformer, str)
+        else (name, transformer, columns)
+        for name, transformer, columns in tv.transformers_
+    ]
+    # Sort to ignore order
+    assert sorted(clean_transformers_) == sorted(expected_transformers_)
```

Comment:
We shouldn't sort before testing because we might run into surprises down the line. Instead, write the final expected output explicitly.

Reply:
Well, I though this the other way around: since the order of the output features is currently quite unstable, and subject to change (#626), sorting them here would make this test independent to the output order.

Reply:
Ok, we'll need another test for stability then